pkg/controller/elasticsearch/name/name.go

```diff
@@ -70,6 +70,9 @@ func Validate(es v1alpha1.Elasticsearch) error {
 
 		// length of the ordinal suffix that will be added to the pods of this sset
 		podOrdinalSuffixLen := len(strconv.FormatInt(int64(nodeSpec.NodeCount), 10))
+		if nodeSpec.NodeCount < 10 {
```

Comment:
I'm not sure if I follow why this is needed. I'd think you always need to account for dash.

Reply:
You are correct. I made the mistake of thinking in round numbers only.